external/akka/src/main/scala/org/apache/spark/streaming/akka/ActorReceiver.scala

```diff
@@ -162,6 +177,18 @@ abstract class JavaActorReceiver extends UntypedActor {
   def store[T](item: T) {
     context.parent ! SingleItemData(item)
   }
+
+  /**
+   * Store a single item of received data to Spark's memory synchronously.
+   * These single items will be aggregated together into data blocks before
+   * being pushed into Spark's memory.
+   *
+   * As opposed to [[ActorReceiver.store[T]: Unit]], this method allows flow control
+   * (maxRate, backpressure) to block the input.
+   */
+  def storeSync[T](item: T)(implicit timeout: Timeout) {
```

Comment:
I assume you are talking about line 190. Removed ```implicit```. akka ```Timeout``` should work with java. See http://doc.akka.io/docs/akka/2.4.2/java/untyped-actors.html#Ask__Send-And-Receive-Future